manifests/plugin/gutterball.pp

```diff
@@ -1,9 +1,26 @@
 # gutterball plugin
 class katello::plugin::gutterball{
+  $foreman_gutterball_gem_config_tmp = '/tmp/foreman_gutterball.yaml'
+
+  $package = "${foreman::plugin_prefix}gutterball"
+  case $::osfamily {
+    'Debian': {
+      $gb_package_name = regsubst($package,'_','-','G')
+    }
+    default: {
+      $gb_package_name = $package
+    }
+  }
+
   Class[ 'certs' ] ->
   class { 'certs::gutterball': } ->
   foreman::plugin { 'gutterball': } ->
   class { '::gutterball':
     keystore_password => $certs::gutterball::gutterball_keystore_password,
+  } ~>
+  class { 'katello::plugin::gutterball::config':
```

Comment:
that would be nice to let configs manage their own configs easily.
I kept it in it's own class so it could be reused by puppet-katello_devel https://github.com/Katello/puppet-katello_devel/pull/30